Sources/TuistGenerator/Mappers/SynthesizedResourceInterfaceProjectMapper.swift

```diff
@@ -182,7 +182,7 @@ public final class SynthesizedResourceInterfaceProjectMapper: ProjectMapping { /
                     }
                 }
             }()
-        case .assets, .coreData, .fonts, .interfaceBuilder, .json, .plists, .yaml, .files:
+        case .assets, .coreData, .fonts, .interfaceBuilder, .json, .plists, .yaml, .files, .stringsCatalog:
```

Comment:
I'd open a separate PR for this one to keep the changes in this PR scoped to the goal of adding support for XCFramework `.dylib` and platforms.

Reply:
I think it's ok to merge this one first without having a separate PR. The `.stringsCatalog` was added in `XcodeGraph` but it's not implemented, yet.

The move to `XcodeGraph` is unfortunately adding a bit of friction as it's easy for changes between `XcodeGraph` and `tuist` to get out of sync :/

Reply:
Yes, it is not relevant to this PR. We can merge this first if implementing the `.stringCatalog` feature is not on track yet, since it not break previous behavior; otherwise, it can wait until that work is completed.